frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/SendNotificationHandler.java

```diff
@@ -369,8 +369,12 @@ private Optional<ErrorResponse> isCodeRequestAttemptValid(
             LOG.info(
                     "User has requested too many OTP codes. Setting block with prefix: {}",
                     newCodeRequestBlockPrefix);
-            codeStorageService.saveBlockedForEmail(
-                    email, newCodeRequestBlockPrefix, configurationService.getLockoutDuration());
+            if (!configurationService.supportReauthSignoutEnabled()) {
```

Comment:
Is this the complete condition? I.e. are we always in reauth journeys at this point? If not, does this not mean when the flag is on we'd not be blocking for all journeys?

Also, if we could have some tests for these scenarios that'd be good :-)

Reply:
These changes aren't needed.  This lambda is only called from the resend-email-code-controller which is not a route for re-authenticating users.